rest/context_test.go

```diff
@@ -97,3 +132,35 @@ func TestHeader(t *testing.T) {
 
 	assert.Equal(req.Header, ctx.Header())
 }
+
+func TestBuildURL(t *testing.T) {
+	assert := assert.New(t)
+
+	api := NewAPI(NewConfiguration())
+	api.RegisterResourceHandler(TestResourceHandler{})
+	api.RegisterResourceHandler(ComplexTestResourceHandler{})
+
+	req, _ := http.NewRequest("GET", "https://example.com/api/v1/widgets", nil)
+	gContext.Set(req, "version", "1")
```

Comment:
Coverage added in https://github.com/aaronkavlie-wf/go-rest/commit/b1bd6a15687b42b039987311c58d3938806c94be